<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shape Creator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBoxStandard">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Standard Options</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Shape:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxPrune">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Prune</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxShape">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Depth:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxDepth">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxResPrune">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Resolution Prune</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxCrunchstore">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>401</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Crunchstore</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DB File Name..:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEditDbFile">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonWriteToDb">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Database</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonWriteToFiles">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>110</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Files</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEditFileFolder">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Output Folder.:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBrowseFolder">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>60</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Output Format:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxVtk">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>611</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>VTK</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>File:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEditVtkFile">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonVtkBrowse">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidgetVtkScalars">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>171</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Scalars:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Vectors:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidgetVtkVectors">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>100</y>
       <width>171</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>110</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Threads:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxVtkCacheOn">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>150</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>cache on</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxVtkHiresLod">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>180</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>HighResLod</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxVtkThreads">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>110</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonVtkClearAll">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear All</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonVtkSelectAll">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Select All</string>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>601</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserOutput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>611</width>
      <height>192</height>
     </rect>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::WidgetWidth</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCreate">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>680</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCancel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>680</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
